corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -91,10 +91,10 @@ hqDefine("app_manager/js/details/case_claim", function () {
             receiverExpression: '',
             itemsetOptions: {},
             exclude: false,
-            required: '',
```

Comment:
I've been thinking about updating this file to use the knockout mapping plugin, which would make the js match the python structure & attribute names exactly. We so often update this file with boilerplate-type code.

Reply:
nice, that'd be cool to move the django view logic there